### DISCUSSION TITLE
Sorted - Always Alphabetically

### DISCUSSION THREAD

**Discussion by t3kg33k** [mealie-recipes/mealie#1432] [General]
Hello,
New to Mealie. Is there a way to set a config so that every time you view All Recipes that the default view is always alphabetically, even if you are not logged in?
Thanks.

**Comment by michael-genson** [comment]
Hey [t3kg33k](https://github.com/t3kg33k), welcome! It sounds like you're on version 0.5.6. At this time we're working towards version 1.0.0, I'd recommend you migrate there if you're able:
https://nightly.mealie.io/documentation/getting-started/migrating-to-mealie-v1/

We actually changed this in the most recent nightly release, in PR #1405. While you're still unable to set your own default, the new default  sort is alphabetical. We're working on some backend changes to get filtering working right, so stay tuned!

**Reply by t3kg33k** [reply]
> Hey [t3kg33k](https://github.com/t3kg33k), welcome! It sounds like you're on version 0.5.6. At this time we're working towards version 1.0.0, I'd recommend you migrate there if you're able: https://nightly.mealie.io/documentation/getting-started/migrating-to-mealie-v1/
> 
> We actually changed this in the most recent nightly release, in PR #1405. While you're still unable to set your own default, the new default sort is alphabetical. We're working on some backend changes to get filtering working right, so stay tuned!

Thanks so much for the response. I'm looking forward to the update.

**Reply by michael-genson** [reply]
No problem! It's merged into nightly now if you want to update, otherwise it'll make its way into one of the betas sooner or later

**Comment by epheterson** [comment]
@michael-genson What's the sort for tags/categories pages? Also could probably use defaulting to alphabetical, even though it already has the sort UI.

BTW updated to nightly and do appreciate alphabetical for All Recipes, thanks!

**Reply by michael-genson** [reply]
Glad to hear! And I'm pretty sure those default to the created at time, but we can consider changing that default sort, too.

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, and the sort options are different, so I'm going to close this
